server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/Requeue.java

```diff
@@ -34,45 +34,82 @@
 import org.apache.mailet.Mail;
 import org.apache.mailet.base.GenericMailet;
 
-public class RequeueMailet extends GenericMailet {
-    private static final MailQueueName DEFAULT_MAIL_QUEUE_NAME = MailQueueFactory.SPOOL;
-    private static final String DEFAULT_PROCESSOR = Mail.TRANSPORT;
-    private static final Duration DEFAULT_DELAY_DURATION = Duration.ofHours(2);
+import com.google.common.base.Preconditions;
 
+/**
+ * <p><b>Requeue</b> allows for a throttling when combined with rate limiting mailet. {@link org.apache.james.transport.mailets.PerSenderRateLimit}</p>
```

Comment:
throttling is a usage not the main function of this mailet.

Say "Requeue puts back the email in a queue.

It can be used for throttling when combined with rate limiting mailet."